client/my-sites/checkout/cart/cart-free-user-plan-upsell.jsx

```diff
@@ -56,48 +58,90 @@ class CartFreeUserPlanUpsell extends Component {
 	};
 
 	getUpgradeText() {
-		const { cart, planPrice, translate } = this.props;
+		const { cart, planPrice, translate, eligibleForProPlan } = this.props;
 		const firstDomain = find( getAllCartItems( cart ), this.isRegistrationOrTransfer );
+		const planName = eligibleForProPlan ? 'Pro' : 'Personal';
 
 		if ( planPrice > firstDomain.cost ) {
 			const extraToPay = planPrice - firstDomain.cost;
-			return translate(
-				'Pay an {{strong}}extra %(extraToPay)s{{/strong}} for our Personal plan, and get access to all its ' +
-					'features, plus the first year of your domain for free.',
-				{
-					args: {
-						extraToPay: formatCurrency( extraToPay, firstDomain.currency ),
-					},
-					components: {
-						strong: <strong />,
-					},
-				}
-			);
+			return eligibleForProPlan
+				? translate(
+						'Pay an {{strong}}extra %(extraToPay)s{{/strong}} for our %(planName)s plan, and get access to all its ' +
```

Comment:
It doesn't look necessary to use `%(planName)` here, since it will be `Pro` for sure by `eligibleForProPlan` holding true. It will also resolve @sirbrillig 's concern :)

Reply:
Fair enough. I moved included the "Pro" plan inside the translation string.